### DISCUSSION TITLE
Feature Request - Ability to search more than one tag at a time

### DISCUSSION THREAD

**Discussion by Pemmican6** [BlafKing/sd-civitai-browser-plus#44] [Ideas]
Apologies if this is already implemented, but I would like to be able to search tags by more than one term at a time.
For instance, instead of just searching for 'Actor', I would like to search by 'Actor' AND 'Man' to further narrow the search.

I tried as many combinations of syntax as I could and I just get no result, so I assume this had not been implemented.
If I am incorrect, please let me know, otherwise, please consider adding multi -term searches.

Thanks

**Comment by BlafKing** [comment]
I'd love to implement this but it looks like the [CivitAI REST API](https://github.com/civitai/civitai/wiki/REST-API-Reference#get-apiv1models) does not support multiple tags in one search.
(Which really should be possible imo)

I've created a feature request [here](https://github.com/orgs/civitai/discussions/746) on the CivitAI Github, feel free to give it an upvote so that it's hopefully implemented soon.
There's unfortunately not much I can do until it has been implemented :/

**Comment by Pemmican6** [comment]
Thanks for the update, I will definitely go give it an upvote, it makes no sense, especially since they now have a multiparameter search on the site.
